### PR TITLE
New Chapter: Monitoring using Amazon Managed Service for Prometheus / Grafana

### PR DESCRIPTION
**Pull request by petrokashlikov**
New module for ecsworkshop

### CODE REVIEW

**Review by adamjkeller — COMMENTED** [review]
Can you add a cleanup section at the end please?

File: content/amp_amg/_index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+{{% button href="https://aws.amazon.com/prometheus/faqs/" icon="fab fa-leanpub" icon="fab fa-leanpub" icon-position="right"  %}}Learn more about AMP{{% /button %}}
+
+
+
+#### Amazon Managed Service for Grafana (AMG)
```
Comment (adamjkeller) [inline content/amp_amg/_index.md:23]: The formatting on this page is a bit funky. Can you take a look and clean up pls?

File: content/amp_amg/software.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Setting CDK Version
+export AWS_CDK_VERSION="1.41.0"
+
+# Install aws-cdk
+npm install -g --force aws-cdk@$AWS_CDK_VERSION
+
+# Install cdk packages
+pip3 install --user --upgrade aws-cdk.core==$AWS_CDK_VERSION \
+aws-cdk.aws_ecs_patterns==$AWS_CDK_VERSION \
```
Comment (adamjkeller) [inline content/amp_amg/software.md:23]: Are all of these modules needed? Can we scope this down?

Comment (petrokashlikov) [reply content/amp_amg/software.md:23]: This is basically platform part to build eco cluster, same as in other modules.

**Comment by petrokashlikov** [comment]
> Can you add a cleanup section at the end please?

Will do

**Pull request merged**